db/db_impl/db_impl.cc

```diff
@@ -407,7 +407,7 @@ Status DBImpl::ResumeImpl(DBRecoverContext context) {
     // during previous error handling.
     if (file_deletion_disabled) {
       // Always return ok
-      EnableFileDeletions(/*force=*/true);
+      EnableFileDeletions(/*force=*/true).PermitUncheckedError();
```

Comment:
Can we add assert or check status here in case if different status is returned  in future,  PermitUncheckedError might ignore it?

Reply:
I will add an assert here, it always return OK.